raycicmd/converter.go

```diff
@@ -150,15 +150,22 @@ func (c *converter) convertPipelineStep(step map[string]any) (
 	envMap := c.envMapCopy()
 	result["env"] = envMap
 
-	var envKeys []string
+	envKeys := make(map[string]struct{})
```

Comment:
are we doing this to deduplicate keys? look like the struct{}{} stuff is not used

Reply:
yes, it is (one of) go's way to define a set.